lib/components/artist/header.js

```diff
@@ -67,6 +67,27 @@ class Header extends React.Component {
       </View>
     );
   }
+
+  descriptiveString() {
+    const artist = this.props.artist;
+    const descriptiveString = (artist.nationality ? artist.nationality : "") + this.birthdayString();
```

Comment:
Instead of the ternary operator, you can also use `||`, because in JS the result value of a conditional expression is returned and in JS `null` (amongst `false`, `0`, and `""` are considered ‘falsy’). i.e.

```js
const descriptiveString = (artist.nationality || "") + this.birthdayString();
```

http://programmers.stackexchange.com/a/82599